app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -140,16 +140,23 @@ class AuthRepositoryTest : ShouldSpec(
 			val currentUser: FirebaseUser = mockk {
 				every { displayName } returns "Marian Kowalski"
 				every { email } returns userEmail
+				every { uid } returns "id"
```

Comment:
@kmaslowiec here is a reason why your test is not working, you are using "id" instead of "test1234567890" which is used in other place:
```
val storageReference: StorageReference = mockk(relaxed = true) {
			every { child("user/$userUid") } returns referenceToUserAvatar
		}
```
so just change `every { uid } returns "id"` to `every { uid } returns userUid` and test will pass :D

Reply:
> @kmaslowiec here is a reason why your test is not working, you are using "id" instead of "test1234567890" which is used in other place:
> 
> ```
> val storageReference: StorageReference = mockk(relaxed = true) {
> 			every { child("user/$userUid") } returns referenceToUserAvatar
> 		}
> ```
> 
> so just change `every { uid } returns "id"` to `every { uid } returns userUid` and test will pass :D

Yep, I figure that out, but thanks.